pkg/test-infra/fixture/fixture.go

```diff
@@ -173,6 +173,12 @@ func init() {
 	flag.StringVar(&Context.LokiUsername, "loki-username", "", "loki username. Needed when basic auth is configured in loki")
 	flag.StringVar(&Context.LokiPassword, "loki-password", "", "loki password. Needed when basic auth is configured in loki")
 
+	flag.StringVar(&Context.ABTestConfig.TiDB1ConfigMap, "abtest.a.tidb-configmap", "", "tidb configmap for cluster a")
```

Comment:
can we remove `abtest.a.xxx` and reuse the xxx defined on fixture.Context as `abtest.a` values? What's your opinion?